plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/ripemd_160/ripemd_160-length-200/ripemd_160-length-200.uplc.expected

```diff
@@ -0,0 +1 @@
+(program 1.0.0 (con bool True))
```

Comment:
JFYI: I advocate for **not** using golden file machinery for programs that are expected to evaluate to a simple value like boolean `True` and instead verify expected result directly in the test code.

Reply:
I'm happy to refactor this out if that is the way to go. Could you please point me to a place where I should put such a test case?

Reply:
> JFYI: I advocate for not using golden file machinery for programs that are expected to evaluate to a simple value like boolean True and instead verify expected result directly in the test code.

I don't think that necessarily applies here.  This is in the conformance tests and by design _all_ of those are golden tests whose input and output are textual UPLC so that they can be run by other evaluators, in particular by the Agda evaluator in `plutus-metatheory`.